BEST_PRACTICES.md

```diff
@@ -195,3 +196,38 @@ var photo: Photo? {
 Completion blocks in networking calls should be returned in the main thread.
 
 Completion blocks should contain the `error` instead of success/failure blocks.
+
+## Theming
+
+Use a `Theme.swift` that contains extensions for theming your app. When adding color extensions avoid adding the `Color` postfix to your color methods.
+
+For example:
+
+```swift
+import Hex
+import UIKit
+
+extension UIColor {
+    class func brightPurple() -> UIColor {
```

Comment:
Depends on how we do it. If we have schemes like [this](https://cloud.githubusercontent.com/assets/54423/14979824/0221c706-1125-11e6-8aaf-c1b2a8e40050.png), it works fine, no?